remoteappmanager/static/js/home/models.js

```diff
@@ -154,18 +153,18 @@ define([
             configurablesData[tag] = configurable.asConfigDict();
         });
 
```

Comment:
create a promise. if you hit any of the fail, you resolve the promise by reject. otherwise, you say success.